src/main/apiClient.ts

```diff
@@ -424,7 +430,8 @@ export default class ApiClient {
               redirectUrl,
               scope: this.resolveScope(auth),
               ...pick(auth, 'origin'),
-              data
+              data,
+              returnToAfterConfirmEmail,
```

Comment:
We hopefully allow trailing commas ? :D

Reply:
No, we don't.

Reply:
We do ! It has never been stated whether we should or should not have trailing coma. IMO there's no need to define whether we use trailing coma or not.

Reply:
But it's actually not the case in the project. If you want to change the global syntax, create a specific PR for that, please.

Reply:
trailing comas FTW !